lib/src/point_on_feature.dart

```diff
@@ -0,0 +1,214 @@
+import 'dart:math' as math;
+import 'package:geotypes/geotypes.dart'; // We still need the GeoJSON types, as they're used throughout the package
+
+/// Returns a Feature<Point> that represents a point guaranteed to be on the feature.
```

Comment:
make sure we are using the proper dart doc comment syntax (wrapping types in [ ] e.g.)

https://dart.dev/effective-dart/documentation